code/modules/projectiles/guns/smgs.dm

```diff
@@ -111,12 +111,13 @@
 	aim_speed_modifier = 0.55
 
 	accuracy_mult = 1.15
-	accuracy_mult_unwielded = 0.9
+	accuracy_mult_unwielded = 0.6
 	scatter = 2
 	fire_delay = 0.15 SECONDS
-	scatter_unwielded = 8
+	scatter_unwielded = 25
```

Comment:
this value is literally way too much absurd, scatter is a value that ranges from ~0 to 45(effective ones that is), 25 scatter unwielded is WAY too much.

Reply:
Again absurd, you're looking to nerf akimbo, not the gun itself. Go nerf the akimbo fire delay.